### PR TITLE
Suppress the `DeprecationWarning` caused by the `trimesh` package

### PR DESCRIPTION
**Pull request by qianyi-sun**
Suppress the ```/root/SMARTS/.venv/lib/python3.8/site-packages/trimesh/curvature.py:12: DeprecationWarning: Please use `coo_matrix` from the `scipy.sparse` namespace, the `scipy.sparse.coo` namespace is deprecated.```  warning.

### CODE REVIEW

**Review by saulfield — APPROVED** [review]

File: smarts/core/scenario.py [hunk 1]
```diff
@@ -66,6 +66,11 @@
 
 VehicleWindow = TrafficHistory.TrafficHistoryVehicleWindow
 
+# Suppress trimesh deprecation warning
+
+with warnings.catch_warnings():
+    warnings.filterwarnings("ignore", category=DeprecationWarning)
+    import trimesh # only suppress the warnings caused by trimesh
```
Comment (Adaickalavan) [inline smarts/core/scenario.py:73]: + Does the catch warning also cater to `import trimesh` in other files, such as `opendrive_road_network.py`, `sumo_road_network.py`, `waymo_map.py`, and `geometry.py`? 
+ Can catch warning be implemented inside `map_glb_meta_for_file()` method below in this file where `trimesh` is being imported?

Comment (Gamenot) [reply smarts/core/scenario.py:73]: @qianyi-sun Could you make the warning more specific? It is possible to pattern match warnings to ignore.

**Review by Adaickalavan — APPROVED** [review]

File: smarts/core/scenario.py (hunk 1 above)
Comment (qianyi-sun) [reply smarts/core/scenario.py]: @Adaickalavan I checked running waymo/opendrive/sumo scenarios, seems like the warning is gone in all three type of scenarios. 
@Gamenot If I understood "specfic" correctly, I just added a `message=` option in the `warnings.filterwarning` to only suppress this specific message.

Comment (Gamenot) [reply smarts/core/scenario.py]: Yes, that is what I meant.

**Review by Gamenot — APPROVED** [review]

**Pull request merged**